src/ThresholdHeuristics.cpp

```diff
@@ -0,0 +1,7 @@
+#include "ThresholdHeuristics.h"
+
+bool ThresholdHeuristics::belongToCluster(
+    libMesh::Elem *elem, libMesh::Elem *neighbor_elem) {
+    return ((getElementDataFromMesh(elem) > _standard_deviation_tolerance &&
```

Comment:
Do we want an option for this to test either greater than or less than the threshold?